packages/exiftool.vm/exiftool.vm.nuspec

```
@@ -2,14 +2,15 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>exiftool.vm</id>
-    <version>13.27.0</version>
+    <version>13.27.0.20250504</version>
     <authors>Phil Harvey</authors>
-    <description>A tool for reeding and writing file metadata</description>
+    <description>Exiftool is tool for reading and writing file metadata.</description>
```

Use the official tool name from the official home page (here and for other tools).
```suggestion
    <description>ExifTool is a tool for reading, writing and manipulating metadata.</description>
```